test/fixtures/cwd.css

```diff
@@ -1,2 +1,2 @@
-@import "test/fixtures/imports/foo.css";
-@import "test/fixtures/imports/foo-recursive.css";
+@import "imports/foo.css";
+@import "imports/foo-recursive.css";
```

Comment:
this test should not be changed since the point of this one is to check that without from option, you can use a file from your cwd (which is the root of this project when running test from the root)